tests/expectations/snapshots/snap_test_expectation_atomic_renderers.py

```diff
@@ -660,15 +656,13 @@
     "value": {
         "params": {
             "column": {"schema": {"type": "string"}, "value": "my_column"},
-            "condition_parser": {"schema": {"type": "string"}, "value": None},
             "mostly": {"schema": {"type": "number"}, "value": 0.8},
             "mostly_pct": {"schema": {"type": "string"}, "value": "80"},
             "parse_strings_as_datetimes": {
                 "schema": {"type": "boolean"},
                 "value": True,
             },
-            "row_condition": {"schema": {"type": "string"}, "value": None},
-            "strictly": {"schema": {"type": "boolean"}, "value": 10},
+            "strictly": {"schema": {"type": "boolean"}, "value": True},
```

Comment:
This was caused by an incorrect test case and should not have been treated as a correct test.